packages/core/admin/server/controllers/admin.js

```diff
@@ -93,11 +94,13 @@ module.exports = {
     const useTypescriptOnAdmin = await isUsingTypeScript(
       path.join(strapi.dirs.app.root, 'src', 'admin')
     );
+    const isHostedOnStrapiCloud = env.bool('STRAPI_CLOUD');
```

Comment:
I don't like having Cloud specific code being added to Strapi. We've always said that we should do this.
I know that this is a temporary solution until the Cloud can do its own telemetry. But still. I feel weird about this.

Reply:
I've reverted to `STRAPI_HOSTING` and used `strapi.cloud` as a value